README.md

```diff
@@ -20,11 +20,27 @@ Example ACL to add a rate limit to the route `/session` of a given ALB (The ALB
 will respond with 403 after 2000 requests per 5 minutes from the same IP).
 
 ```hcl
+local {
+  allowed_headers = [
+    {
+      target_string = "sample-string-1"
+      positional_constraint = "STARTS_WITH"
+      text_transformation = "NONE"
+      type = "HEADER"
+      data = "referer"
+    },
+    {
+      target_string = "sample-string-2"
+      positional_constraint = "EXACTLY"
+      text_transformation = "NONE"
+      type = "URI"
+    }
+  ]
+}
 module "rate_based_rule" {
   source                = "github.com/ryte/INF-tf-waf.git//rate_based_rule?ref=v0.3.2"
-  pattern               = "/session"
-  positional_constraint = "STARTS_WITH"
-  metric_name           = "WAFRBRuleMatchSession"
+  allowed_headers = local.allowed_headers
+  metric_name     = "WAFRBRuleMatchSession"
```

Comment:
Please fix the indentation manually until we have a pre-commit hook implemented for this repo